dashboard/app/models/unplugged.rb

```diff
@@ -24,4 +24,32 @@
 #
 
 class Unplugged < Level
+  def self.create_from_level_builder(params, level_params)
+    create!(level_params.merge(user: params[:user], game: Game.unplugged, level_num: 'custom'))
+  end
+
+  def assign_attributes(new_attributes)
+    i18n_strings = {
+      'title' => new_attributes.delete(:title),
+      'desc' => new_attributes.delete(:description),
+    }
+    update_i18n(new_attributes[:name], i18n_strings)
+
+    super(new_attributes)
+  end
+
+  def title
+    I18n.t("data.unplugged.#{name}.title")
+  end
+
+  def description
+    I18n.t("data.unplugged.#{name}.desc")
+  end
+
+  def update_i18n(name, new_strings)
+    unplugged_yml = File.expand_path('config/locales/unplugged.en.yml')
+    i18n = File.exist?(unplugged_yml) ? YAML.load_file(unplugged_yml) : {}
+    i18n.deep_merge!({'en' => {'data' => {'unplugged' => {name => new_strings}}}})
+    File.write(unplugged_yml, i18n.to_yaml(line_width: -1))
+  end
```

Comment:
I followed the [pattern you created in `Script.update_i18n`](https://github.com/code-dot-org/dashboard/commit/bc32f2ed9df37b69b5edb7fd6fca00a4942d1f49#diff-9cacd63540753d6929377d6cb58210cbR192) 😄.

I'd like to understand the Rails I18n backend pipeline a little better, then possibly switch this to use `I18n.backend.store_translations`.  Probably a future PR!